src/pages/trello/index.tsx

```diff
@@ -1,10 +1,68 @@
+import { useEffect, useState } from 'react';
+import CardDetailsList, { DataProps } from '../../components/card-detail-list';
 import Container from '../../components/container';
 import './trello.css';
+import dummyDatas from '../../json/dummyTrello.json';
 
 const Trello = () => {
+  const [data, setData] = useState<DataProps[]>([]);
+
+  useEffect(() => {
+    setData(dummyDatas);
+  }, []);
+
+  const onlyStatusAddInArr = (data: DataProps[]) => {
+    const setStatus = new Set();
+
+    for (const item of data) {
+      setStatus.add(item.status);
+    }
+    return Array.from(setStatus);
+  };
+
+  const multipleData = (data: DataProps[]) => {
+    const multiData = [];
+
+    const statusArr = onlyStatusAddInArr(data);
+
+    for (let i = 0; i < statusArr.length; i++) {
+      const element = statusArr[i];
+      multiData.push({
+        id: i,
+        title: element,
```

Comment:
title: element şu sen yeni function javascript ilk büyük harf ve _ olursa boşluk olsun diye yapmalı.
label={
              item.title === 'todo'
                ? 'To Do '
                : item.title === 'in_progress'
                ? ' In Progress'
                : 'Done'
            }
Bu yaptığında ama ilerideki trello gibi yeni daha kart eklenecek olursa. Örneğini 10 tane title nasıl yapacaksınız.
Function daha kolay olur

Reply:
bunu hallettim  commit attım